src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -204,19 +239,19 @@ void RhConfig::ReadEmbeddedSettings(void *volatile * embeddedSettings, void* com
 {
     if (*embeddedSettings == NULL)
     {
-        uint32_t size = ((CompilerEmbeddedSettingsBlob*)compilerEmbeddedSettingsBlob)->Size;
+        uint32_t count = ((CompilerEmbeddedSettingsBlob*)compilerEmbeddedSettingsBlob)->Count;
```

Comment:
We're treating the size as count incorrectly in rest of the code (missed that in the code review). So this is also a bugfix.

Reply:
Ended up deleting all of this.

It ran into an issue that we had a hard limit on how long a key or value can be and instead of writing more C code, I just deleted it and update the compiler to emit it in a format that we don't need to parse at all and just use as-is.